Sources/StreamChat/Controllers/CurrentUserController/CurrentUserController_Tests.swift

```diff
@@ -122,16 +127,21 @@ final class CurrentUserController_Tests: StressTestCase {
         env.currentUserObserverItem = .mock(id: .unique)
 
         var synchronizeError: Error?
+        var synchronizeCalled = false
         controller.synchronize { [callbackQueueID] error in
             AssertTestQueue(withId: callbackQueueID)
             synchronizeError = error
+            synchronizeCalled = true
         }
 
         // Simulate connection not successful
         client.simulateProvidedConnectionId(connectionId: nil)
 
-        XCTAssertEqual(controller.state, .remoteDataFetchFailed(.ConnectionNotSuccessful()))
-        XCTAssertNotNil(synchronizeError)
+        AssertAsync {
+            Assert.willBeEqual(self.controller.state, .remoteDataFetchFailed(.ConnectionNotSuccessful()))
+            Assert.willBeEqual(synchronizeError as? ClientError, ClientError.ConnectionNotSuccessful())
+            Assert.willBeTrue(synchronizeCalled)
```

Comment:
This is redundant because if `synchronizeError` is non-nil it means the synchronize block got called :)

Reply:
✅ Done